connaisseur/alert.py

```diff
@@ -110,20 +110,23 @@ def _get_headers(receiver_config):
 
 def load_config():
     try:
-        alert_config_dir = f'{os.getenv("ALERT_CONFIG_DIR")}'
+        alert_config_dir = f'{os.getenv("ALERT_CONFIG_DIR", "/app")}'
         with safe_path_func(
             open, alert_config_dir, f"{alert_config_dir}/alertconfig.json", "r"
         ) as configfile:
             alertconfig = json.load(configfile)
         schema = get_alert_config_validation_schema()
         json_validate(instance=alertconfig, schema=schema)
     except Exception as err:
-        raise ConfigurationError(
-            "Alerting configuration file either not present or not valid."
-            "Check in the 'helm/values.yml' whether everything is correctly configured. {}".format(
-                str(err)
-            )
-        ) from err
+        if isinstance(err, FileNotFoundError):
+            return {}
```

Comment:
I guess this should be logged, probably info/warning?

Reply:
yeah, I thought about it too - what would you log in that  case? That you have updated but without the new setup? Or just that it's missing - but that might be confusing bc this PR is to ensure everything works just as well with new new image.

Reply:
`Info: No alerting configuration file found. No alerts will be issued` shouldn't be that confusing I'd hope 🤷

Reply:
Hmya, but if you built it with the new templates there might be a alerting configuration file but still no alerts will be issued :D

Reply:
went for a mixture ;-)